index.md

```diff
@@ -335,6 +334,7 @@ includes:
     sufficient contrast;
 -   Text is resizable up to 200% without losing information, using a
     standard browser;
+-   Text spacing can be adjusted without losing information, and reflows on small screens;
```

Comment:
To clarify: The suggestion is to **replace/delete** these two bullets:
-   Text is resizable up to 200% without losing information, using a standard browser;
-   Text spacing can be adjusted without losing information, and reflows on small screens;

With:
- When users resize text up to 400% or change text spacing, no information is lost.
- In small windows ("viewports") or when users zoom text larger, text reflows so that users don't have to scroll to read a line of text.

Reply:
In discussion with @slhenry:

> Text reflows in small windows or when users make text larger.